shared/src/metabase/types.cljc

```diff
@@ -104,6 +104,10 @@
 ;; different from `java.time.OffsetDateTime`;
 (derive :type/Instant :type/DateTimeWithLocalTZ)
 
+
+;; TODO: remove these. They shouldn't be allowed as semantic types. But migrations use them targeting version
+;; 0.20. Not clear if we should remove the tests or keep these forever
+
```

Comment:
I don't want to end up keeping things forever -- our code is complicated enough without a bunch of legacy cruft hanging around. I vote to remove them